### ISSUE TITLE
[Discussion] Update `latest` tag to use MSBuild

### ISSUE THREAD

**Issue by natemcmaster** [aspnet/aspnet-docker#199]
_The `microsoft/dotnet` image will also be making this change. See https://github.com/dotnet/dotnet-docker/issues/201_

Discussion issue for https://github.com/aspnet/Announcements/issues/219

**Comment by DMWOO7** [comment]
It's a good idea to promote the new MSBuild. But I think, that's really too early! When you're using the full Visual Studio IDE in Windows, Version 2017 is required. VS 2017 isn't at least finally released yet, only the RC. Even thought it is, we should give the people try to update. Especially paid licenses wouldn't do an upgrade one day after the final release, since they've to pay the full price. 

In short: We should do this, but not by now. 

Additionally, what's about templates? Are the Visual Studio 2017 templates for ASP.NET Core updated yet, so that new projects would use MSBuild as default configuration format? That would be a step that imho should be done within the release of VS 2017.

**Comment by natemcmaster** [comment]
As the announcement said, the project.json build images still exist. Use `microsoft/aspnetcore-build:1.0.0-projectjson` and `microsoft/aspnetcore-build:1.1.0-projectjson`

VS 2017 templates _only_ use MSBuild SDK and do not support the project.json format.

We realize there are pros and cons of doing it now. When we considered all factors, we decided to move forward with the change. Some of those factors have been explained here: https://github.com/dotnet/dotnet-docker/issues/201

**Comment by natemcmaster** [comment]
Visual Studio 2017 and .NET Core 1.0.0 tooling has RTMed, so project.json images have been discontinued. We won't remove the tagged images, but they will not longer receive updates.
